benchmark/pyproject.toml

```diff
@@ -9,44 +9,46 @@ packages = [{ include = "agbenchmark" }]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-pytest = "^7.3.2"
-requests = "^2.31.0"
-openai = "^1.7.2"
-pydantic = "^1.10.9"
-python-dotenv = "^1.0.0"
+# helicone = "^1.0.9"  # incompatible with openai@^1.0.0
+agent-protocol-client = "^1.1.0"
 click = "^8.1.3"
-types-requests = "^2.31.0.1"
-pexpect = "^4.8.0"
-psutil = "^5.9.5"
-matplotlib = "^3.7.2"
-pandas = "^2.0.3"
+click-default-group = "^1.2.4"
+colorama = "^0.4.6"
+fastapi = "^0.109.1"
 gitpython = "^3.1.32"
+httpx = "^0.24.0"
+matplotlib = "^3.7.2"
 networkx = "^3.1"
-colorama = "^0.4.6"
-pyvis = "^0.3.2"
-selenium = "^4.11.2"
+openai = "^1.7.2"
+pandas = "^2.0.3"
+pexpect = "^4.8.0"
+psutil = "^5.9.5"
+pydantic = "^1.10.9"
+pytest = "^7.3.2"
 pytest-asyncio = "^0.23.3"
-uvicorn = "^0.23.2"
-fastapi = "^0.109.1"
+python-dotenv = "^1.0.0"
 python-multipart = "^0.0.7"
-toml = "^0.10.2"
-# helicone = "^1.0.9"  # incompatible with openai@^1.0.0
-httpx = "^0.24.0"
-agent-protocol-client = "^1.1.0"
-click-default-group = "^1.2.4"
+pyvis = "^0.3.2"
+requests = "^2.31.0"
+selenium = "^4.11.2"
 tabulate = "^0.9.0"
+toml = "^0.10.2"
+uvicorn = ">=0.23.2,<1"
```

Comment:
Why no 1.0 support?

Reply:
Because that will almost certainly be a breaking update